src/test/scala/com/fullcontact/interview/RecordFinderTest.scala

```diff
@@ -1,9 +1,26 @@
+/*
+Author: FullContact
+Modified By: Jeff Willingham
+Creation Date: unknown
+Modification Date: 2020-07-28
+Purpose: This script checks Queries.txt, Records.txt and RecordFinder.scala to determine if they (1) exist and (2) contain data.
+*/
+
 package com.fullcontact.interview
 
 import org.scalatest.{FunSuite, Matchers}
+import java.io.File
 
 class RecordFinderTest extends FunSuite with Matchers {
```

Comment:
Following Jing's question, how would you generally approach testing code, including unit tests and other means?